### ISSUE TITLE
Unrecognised EGLO Devices ?

### ISSUE THREAD

**Issue by wickedgalen** [fsaris/home-assistant-awox#46]
Hi to all,

I recently added my Eglo Lights into HACS, essentially all good and happy about it.

Although 2 Lights are not appearing for now, unsure if the error is with me or if there is an issue of some kind with your Module (not knowing what needs are there to integrate the different lamp types).

EFueva_300r (Version 2.2.6 (HW 1.0))
ECeil_G30 (Version 2.2.6 (HW 1.0))

I will reset the lamps just in case, the AWOX App on Android recognised them and they both work, just not appearing in HACS.

Any other Ideas ? What other steps can I do?

Thank you.

**Comment by fsaris** [comment]
Enable debug and check the logs for the type of the devices. I expect that it are lights working on a unsupported protocol

https://github.com/fsaris/home-assistant-awox#troubleshooting

**Comment by wickedgalen** [comment]
Hi @fsaris : 
I managed to get it working.

In the logs was no error to be related to "custom_components.awox".
The "missing" lights were not listed.

I resolved my issue by : Disabling, Uninstall and Reinstalling your component. 
This, I guess, refreshed the light list properly and all Lights are showing now and can be controlled.

Thereafter only, I took a closer look on your ToDo list and I suppose for whatever reason I did not save the light properly in the Awox app or something !?!

Thanks for your time and effort to make this component =)
